javers-core/src/test/groovy/org/javers/core/diff/appenders/OptionalChangeAppenderTest.groovy

```diff
@@ -50,9 +52,12 @@ class OptionalChangeAppenderTest extends AbstractDiffAppendersTest {
         ChangeAssert.assertThat(change)
                     .hasInstanceId(SnapshotEntity, 1)
         change instanceof ReferenceChange
-        change.left == expectedLeftRef
-        change.right == expectedRightRef
-
+        ReferenceChangeAssert.assertThat( change )
+            .hasLeftReference( expectedLeftRef )
+            .hasRightReference( expectedRightRef )
+            .hasLeftObject( org.javers.common.collections.Optional.empty() )
+            .hasRightObject( org.javers.common.collections.Optional.empty() )
```

Comment:
i mean why you expect that both leftObject and rightObjects are Optional.empty() ?